Cargo.toml

```diff
@@ -48,7 +48,6 @@ tera = ["dep:tera", "dep:heck", "dep:serde_json"]
 
 [dependencies]
 handlebars = { version = "5", optional = true }
-fluent = "0.16"
 fluent-bundle = "0.15"
 fluent-syntax = "0.11"
 fluent-langneg = "0.13"
```

Comment:
It might be better to keep `fluent`, drop the other related crates, and use the re-exported modules directly from fluent. That has a few advantages including making it easier to keep them in sync. For example it isn't immediately obvious why *fluent-langneg* is held back from the latest version here if you don't understand that's because the types in the new release don't match the ones expected by other crates that haven't updated ICU stuff yet. Using the *fluent* crate pushes that complexity off for upstream to sort out and depend on the right versions.

Reply:
The problem is that `fluent` only includes `fluent-bundle`, `fluent-pseudo` and `unic-langid` as dependencies, so we can't use it. See https://crates.io/crates/fluent/0.16.1/dependencies Currently, this dependency is only making fluent-templates to compile a dependency (`fluent-pseudo`) that is not needed.

It seems that `fluent` is designed for end users.

Reply:
> The problem is that `fluent` includes `fluent-bundle`, `fluent-pseudo` and `unic-langid` as dependencies. See [crates.io/crates/fluent/0.16.1/dependencies](https://crates.io/crates/fluent/0.16.1/dependencies) Currently, `fluent` is only making fluent-templates to compile a dependency (`fluent-pseudo`) that is not needed. Altough we can depend on `fluent-bundle`, `fluent-pseudo` will be compiled anyway.

Fair enough. I think everything you don't need will be compiled away anyway, but it *does* add crates you aren't using to the dependency tree (and SBOM etc.) and not wanting that baggage is a good enough reason to be more targeted.

Frankly the `fluent` crates layout is a bit wonky. As best I can figure it changed directions a couple times during development and never really got cleaned up after the direction solidified.

This does make me think we need to consider whether more targeted features. It's kind of awkward that there are two completely different ways to access everything with different (and kind of unclear) pros and cons to each way.

> It seems that `fluent` is designed for end users.

What is an "end user"?